pandas/core/frame.py

```diff
@@ -4341,27 +4341,31 @@ def last_valid_index(self):
 
     def pivot(self, index=None, columns=None, values=None):
         """
+        Return reshaped DataFrame summarized by given index / column values.
+
         Reshape data (produce a "pivot" table) based on column values. Uses
         unique values from index / columns to form axes of the resulting
-        DataFrame.
+        DataFrame. This function does not support data aggregation, multiple
+        values will result in hierarchically indexed columns.
 
         Parameters
         ----------
         index : string or object, optional
             Column name to use to make new frame's index. If None, uses
```

Comment:
Would it be clearer to just say "Column" instead of "Column name". I know the value you specify is the name, but what you actually use to make the new frame's index is the actual column, not the column name ...